packages/gui/src/components/settings/SettingsNFT.tsx

```diff
@@ -33,6 +115,48 @@ export default function SettingsGeneral() {
               }
               label={<Trans>Hide objectionable content</Trans>}
             />
```

Comment:
The "Hide objectionable content" switch isn't hooked up yet. Can you comment it out?